### DISCUSSION TITLE
Why the 13 year minimum age?

### DISCUSSION THREAD

**Discussion by ausmark** [community/community#44742] [GitHub Education] [answered]
Our child starts high school this year, and since early last year has learned a lot about programming and would like to continue.

A friend referred us to GitHub Education, only our child isn't old enough yet.

In Australia, it's not unusual to begin high school at 12. I'm just curious, why is there a minimum age limit that restricts a lot of kids? Our child is very keen to learn, and has already done a lot of self-driven learning (Unity/C#, Roblox/Lua/client-server programming, Discord/python/embedded SQL/IPC, HTML/CSS/javascript, etc), so we want them to support their learning as best we can, and GitHub Education seems ideal -- apart from the age restriction.

Is there a way we can get our child signed up to this?

Thanks for your help.

**Comment by VoHoAnhTuan** [comment, marked as answer]
https://docs.github.com/en/site-policy/github-terms/github-terms-of-service

I found this link and in section 3, they say they must comply with US law. Unfortunately, they don't reference it specifically. I hope this is helpful to you.

**Reply by ausmark** [reply]
Thank you for that. Interesting. Yeah it's pretty clear it's to comply with US law, which is fair enough for US residents. They go on to say that some countries may have an older minimum age, but nothing about younger. Still...

I think we'll just have to wait for 13 to roll along... unless anyone else knows anything different?

**Reply by nikzeyn** [reply]
There are a lot that are not unusual in Australia, but not in the other hemi sphere :)

The exact same issue with us - my kids wanted to use GitHub but they could not due to the age restrictions. At the end they decided to use their own local git repo for now until they are 13 :)

I wish GitHub would introduce a parental control feature  - something like Microsoft/Google have.

**Reply by nosh-ii** [reply]
For your information, this whole age limit thing is quite a slippery slope.

Although, you are only required to be over 13 to **own** the account. The child may still use the account, but the adult must own it. _**PLEASE CORRECT ME IF I AM WRONG, FELLOW GITHUBERS!**_

**Reply by SollyBunny** [reply]
> For your information, this whole age limit thing is quite a slippery slope.
> 
> Although, you are only required to be over 13 to **own** the account. The child may still use the account, but the adult must own it. _**PLEASE CORRECT ME IF I AM WRONG, FELLOW GITHUBERS!**_

Although accounts created by bots are banned, bot accounts are allowed if the account is created (and owned) by a human. The same rules could apply to children

Either way, I don't think the rule has or ever will be enforced since there's no reason to do so

**Reply by rlalik** [reply]
It is most like due to COPPA -  https://en.wikipedia.org/wiki/Children's_Online_Privacy_Protection_Act

There is a button below to Close the thread, please use it if you think your problem was solved.
